testslide/mock_callable.py

```diff
@@ -462,7 +462,8 @@ def _get_runner(self, *args, **kwargs):
 
     def _validate_return_type(self, runner, value):
         if runner.TYPE_VALIDATION and runner.original_callable is not None:
-            _validate_return_type(runner.original_callable, value)
+            _validate_return_type(runner, value)
```

Comment:
If you send runner here,, it'll check value against `_CallableMock.__call__`, which returns `Any`, so I think nothing will be checked.

Reply:
I dont think so or im missing something, i also changed the _validate_return_type function to take in to account runner is being sent in there

Reply:
It did break a mock_callable test, check Travis CI. Just run `make` on your machine to run all tests before pushing.

PS: signal in Travis CI is hidden because of [this issue](https://github.com/facebookincubator/TestSlide/issues/182).